packages/sfc-playground/src/Header.vue

```diff
@@ -149,13 +153,8 @@ async function fetchVersions(): Promise<string[]> {
       >
         <Download />
       </button>
-      <button title="View on GitHub" class="github">
-        <a
-          href="https://github.com/vuejs/core/tree/main/packages/sfc-playground"
-          target="_blank"
-        >
-          <GitHub />
-        </a>
+      <button title="View on GitHub" class="github" @click="openGithub">
```

Comment:
Using button instead of a tag to open a new page is not conducive to SEO, maybe you need to rethink how to solve it

Reply:
The source code href on github really need to SEO?  😲😲